### ISSUE TITLE
Accessory removed. Then, cant add it anymore

### ISSUE THREAD

**Issue by davidino** [KhaosT/homebridge-camera-ffmpeg#219]
Hi all, 
everything was working fine, but I had to remove the camera from the accessories. Never able to add it again.

This is my setup: 
`MotionEye` running on `192.168.0.44` on Raspberry pi 0
`Homebridge` running on `192.168.0.17` on Raspberry pi 3b

Homebridge configuration:
```json
  "platforms": [{
    "platform": "Camera-ffmpeg",
    "cameras": [{
      "name": "Camera1",
      "videoConfig": {
        "source": "-re -f mjpeg -i http://192.168.0.44:8081",
        "stillImageSource": "-f mjpeg -i http://192.168.0.44/picture/1/current/",
        "maxStreams": 2,
        "maxWidth": 800,
        "maxHeight": 600,
        "maxFPS": 15,
	"debug": false
      }
    }]
  }]
```
When running the logs reports:

```
[10/21/2018, 4:15:57 PM] Homebridge is running on port 51826.
[10/21/2018, 4:15:57 PM] Camera1 is running on port 35073.
```

But then I cannot find the camera in the accessory list. Even clicking on the `+` button ad adding the code.
I've tried to delete the internal folders of `homebridge`, like `persistence` and `accessory` but did not work.

is the problem on:
*  the camera configuration ?
* home app ? 
* remove/add accessory ? 

Can you suggest a way to debug and isolate the issue ?

Thanks for your help.

**Comment by elkorleone** [comment]
same issue for me. It doesn't appear in Home APP

[2018-10-23 14:41:03] Homebridge is running on port 51826.
[2018-10-23 14:41:03] CameraIP is running on port 35767.

**Comment by Samfox2** [comment]
Change the camera name and try again...

**Comment by davidino** [comment]
I finally solved it.

In the configuration i've changed: `maxWidth`, `maxHeight`, `maxFPS`. 
Added `vcoded`.

Final config:
```json
      "videoConfig": {
        "source": "-re -f mjpeg -i http://192.168.0.44:8081",
        "stillImageSource": "-f mjpeg -i http://192.168.0.44/picture/1/current/",
        "maxStreams": 2,
        "maxWidth": 1280,
        "maxHeight": 800,
        "maxFPS": 10,
	"vcoded": "h264",
	"debug": true
      }
```

According exactly to what is reported in the [wiki](https://github.com/KhaosT/homebridge-camera-ffmpeg/wiki/Tested-Configurations) for `motioneye`.

I was not expecting that, but actually the resolution and FPS it can be the reason why you dont find the accessory. I was playing around with all these config before ( also vcodec ) but only this one is actually working.
